### ISSUE TITLE
Udacity Course Research

### ISSUE THREAD

**Issue by mbarlow12** [phetsims/a11y-research#61]
In my recent reading about a11y best practices, I came across this Udacity course on web accessibility:

https://www.udacity.com/course/web-accessibility--ud891

I'm about halfway through and thought I'd make an issue for it. It might be helpful as a rapid way to get new hires/project members up to speed on this type of work.

**Comment by jessegreenberg** [comment]
This is really cool, thanks @mbarlow12! @terracoda are you familiar with this course?

**Comment by terracoda** [comment]
I am familiar with many older courses, but not specifically the Udacity one. I can review it if we want to use it as a trainging tool.

TeachAccess.org is relatively new initiative that has a quick tutorial. It may be a good compliment, or a quick resource.
https://teachaccess.github.io/tutorial/

**Comment by mbarlow12** [comment]
I was planning to report back here once I'd finished it.

**Comment by terracoda** [comment]
About 3 (maybe more) years ago, I tried to do a Google Web Accessibility course, but it forced you to use Google Chrome, which I thought was a bit self-serving. The Udacity course is a google course, so it could be a improved version :-)

**Comment by terracoda** [comment]
I am familiar with Rob Dodson's resources on YouTube. He has made many solid tutorials on web accessibility techniques, so it should be a good course.

**Comment by terracoda** [comment]
@mbarlow12, @jessegreenberg , and @zepumph here's my super brief interpretation of WCAG [Ramp it up!](http://terracoda.ca/ramp/)

**Comment by mbarlow12** [comment]
Finished the content a few days ago. Here are some of the highlights:
- Provided some fantastic insight into how some AT users actually interact with screen readers and keyboard input
- Solid overview of the accessibility tree
- Thorough explanation of the various ARIA features
- Offered some interesting new use cases that raise a11y concerns

Overall, I'd highly recommend this to any devs interested in contributing, and it provides a common base level competency and fluency in the general practices and terminology. You can also pretty easily speed the videos up to consume less time.

**Comment by terracoda** [comment]
Awesome! Glad you liked it. @mbarlow12.

**Comment by mbarlow12** [comment]
Unassignigning myself but keeping issue open for future reference.

**Comment by mbarlow12** [comment]
Leaving open, but unassigning myself. I think this would be useful in the future if we're pulling on new employees/interns who don't have much experience in a11y.

**Comment by zepumph** [comment]
Thanks for making this issue @mbarlow12, I'm excited to poke around in the course.
